src/consensus/tx_verify.cpp

```diff
@@ -20,6 +20,15 @@ bool IsFinalTx(const CTransaction &tx, int nBlockHeight, int64_t nBlockTime)
         return true;
     if ((int64_t)tx.nLockTime < ((int64_t)tx.nLockTime < LOCKTIME_THRESHOLD ? (int64_t)nBlockHeight : nBlockTime))
         return true;
+
+    // Even if tx.nLockTime isn't satisfied by nBlockHeight/nBlockTime, a
+    // transaction is still considered final if all nSequence values are final,
```

Comment:
nit: You could make clear that by final you mean if the nSequence values are maxed out to `0xffffffff`. Perhaps this wording of final is clear to everyone else (in which case ignore).